PlayCover/Views/App Views/StoreAppView.swift

```diff
@@ -28,15 +29,24 @@ struct StoreAppView: View {
                                 isList: isList,
                                 warningSymbol: $warningSymbol,
                                 warningMessage: $warningMessage)
-        .gesture(TapGesture(count: 2).onEnded {
-            if let url = URL(string: app.link) {
-                downloadApp(url, app)
-            }
-        })
-        .simultaneousGesture(TapGesture().onEnded {
-            selected = app
-        })
         .environmentObject(downloadVM)
+        .contextMenu {
```

Comment:
I'm not sure a context menu is the best UI choice here. The cancel button should be more visible i.e. on the download toast itself.